cosmos-sdk-proto/src/lib.rs

```diff
@@ -5,8 +5,7 @@
 //! worth considering that modules seem to show up and go away with every RC.
 //! TODO actually implement features tag based compilation
 
-#![deny(warnings, trivial_casts, trivial_numeric_casts, unused_import_braces)]
```

Comment:
I'm a fan of allowing warnings locally, but enforcing code is warning-free in CI. See the corresponding change here:

https://github.com/cosmos/cosmos-rust/pull/3/files#diff-73e17259d77e5fbef83b2bdbbe4dc40a912f807472287f7f45b77e0cbf78792dR15